hadoop-cloud-storage-project/hadoop-huaweicloud/src/site/markdown/index.md

```diff
@@ -24,12 +24,14 @@ This support comes via the JAR file `hadoop-huaweicloud.jar`.
 
```

Comment:
can you update `overview` section also like below..?

![image](https://user-images.githubusercontent.com/1954406/110891582-4d0bba00-8318-11eb-8f34-cade8b94ba49.png)